README.md

```diff
@@ -41,8 +53,8 @@ Default override arguments:
 - `partial_corpus_filepath`: where to create / load partial corpus file. Default is `''` for creating partial corpus under 'data' folder.
 - `partial_corpus_line_number`: number of lines for partial corpus - in thousands. Default is `1000`.
 - `max_len`: max length of tokens in bytes. Default is `16`.
-- `workers`: number of worker threads to use. Default is `max(1, mp.cpu_count()-1)`.
-- `random_seed`: random seed value. Default is `random.randint(1, 1000)`.
+- `workers`: number of worker threads to use. Default is `1`.
+- `random_seed`: random seed value. Default is `1234`.
```

Comment:
Why is the default now a specific number?

Reply:
Needed to align between execution via shell/main vs via direct usage from Python. As discussed will change random seed to 692653.